docs/apache-airflow/concepts/timetable.rst

```diff
@@ -48,3 +48,58 @@ the schedule. Some examples are:
 As such, Airflow allows for custom timetables to be written in plugins and used by
 DAGs. An example demonstrating a custom timetable can be found in the
 :doc:`/howto/timetable` how-to guide.
+
+Built In Timetables
+-------------------
+
+Airflow comes with several common timetables built in to cover the most common use cases. Additional timetables
+may be available in provider packages.
+
+CronDataIntervalTimetable
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Set schedule based on a cron expression. Can be selected by providing a string that is a valid
+  cron expression to the ``schedule_interval`` parameter of a DAG as described in the :doc:`/concepts/dags` documentation.
+
+.. code-block:: python
+
+    @dag(
+        schedule_interval="0 1 * * 3"  # At 01:00 on Wednesday.
+        ...
+    )
+
+DeltaDataIntervalTimetable
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Schedules data intervals with a time delta. Can be selected by providing a
+  ``datetime.timedelta`` or ``dateutil.relativedelta.relativedelta`` to the ``schedule_interval`` parameter of a DAG.
```

Comment:
We can use ``:class:`datetime.timedelta` `` since we already have intersphinx set up for CPython. We should probably do the same for dateutil as well (maybe in a separate PR).